sphinx/tutorial/rose/furthertopics/polling.rst

```diff
@@ -125,19 +125,20 @@ run the suite to confirm it still works.
 Available Polling Types
 -----------------------
 
-test and all-files are just two of the available polling options:
+Test and all-files are just two of the available polling options:
 
 ``all-files``
-   A list of space delimited list of files which only passes if all file
-   paths in the list exist.
+   Tests if all of the files in a list exist.
 ``any-files``
-   A list of space delimited list of files which passes if any file path in
-   the list exists.
+   Tests if and of the files in a list exist.
 ``file-test``
-   Allows you perform tests on a file if checking its existence is not enough
-   e.g. perform a ``grep``.
+   Changes the test used to evaluate the ``any-files`` and ``all-files`` lists
+   to a shell script to be run on each file (e.g. ``grep``). Passes if the
+   command exits with a zero return code.
 ``test``
-   A shell command which passes if the command returns a 0 (zero) return code.
+   Tests using a shell script, passes if the command exits with a zero return
+   code. *Note this is separate from the ``all-files``, ``any-files`` testing
```

Comment:
The double back-ticks are not formatting as desired within the italicised section, just being printed out as symbols in italics. Try italicising only the text and comma surrounding the keywords?